ape_safe/accounts.py

```diff
@@ -204,17 +219,17 @@ def alias(self) -> str:
         return self.account_file_path.stem
 
     @property
-    def account_file(self) -> dict:
-        return json.loads(self.account_file_path.read_text())
+    def account_file(self) -> SafeCacheData:
+        return SafeCacheData.model_validate_json(self.account_file_path.read_text())
```

Comment:
One thing to keep in mind when reading and writing text files: if you don't specify the encoding, it uses the local preffered one, which is not always utf8 on some systems. For files managed by Ape, I like to just force it to use utf8 so it is consistent across devices and more performant

Reply:
edc73f5